doc/lua_api.txt

```diff
@@ -5384,8 +5421,8 @@ Definition tables
         ^ Name of dropped node when dug. Default is the node itself.
         ^ Alternatively:
         drop = {
-            max_items = 1,  -- Maximum number of items to drop.
-            items = {  -- Choose max_items randomly from this list.
+            max_choices = 1,  -- Maximum number of items to drop.
+            choices = {  -- Choose max_choices randomly from this list.
```

Comment:
Why this change? Items as in ItemStacks is a correct term here.

Reply:
Some people (including Shara) have commented that having items inside items gives them headaches.  Actually, the comment on 5424 is not quite right - it is the maximum number of choices that will be made.  If sand is chosen, there will be 2 item strings returned.

Reply:
Doesn't necessarily mean it should be changed. How many mods will this affect?